stable/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 8.2.1
+version: 8.2.2
```

Comment:
Please bump the minor version since the chart interface has been updated

Reply:
/solved